### ISSUE TITLE
ClientError affiché pour des 404

### ISSUE THREAD

**Issue by XavierJp** [etalab/annuaire-entreprises-site#957]
Peut etre un bug :

ClientErrorPageDisplayed :

https://errors.data.gouv.fr/organizations/sentry/issues/135327/events/?project=15&referrer=issue-stream

**Comment by johangirod** [comment]
Plus précisément, le bug apparait au clique du bouton « copier » quand la traduction automatique de la page par le navigateur est activée.

**Comment by XavierJp** [comment]
Interessant. Est ce que tu arrives a reproduire ? 

Est ce que le bug "casse" le copier coller ? Ou c'est seulement du bruit ?

**Comment by johangirod** [comment]
> Interessant. Est ce que tu arrives a reproduire ?

Oui, facilement, avec browserstack (sur edge, cliquer sur le bouton en haut à droite du navigateur « traduire la page »)

> Est ce que le bug "casse" le copier coller ? Ou c'est seulement du bruit ?

Non, ça casse vraiment le site avec la page d'erreur qui s'affiche (cohérent avec ce qui remonte dans sentry).
